### PR TITLE
Docker Azure deployment action

### LINKED ISSUE #39: Add Azure build and push to ACR workflow
Add Azure build and push to ACR workflow

### PR DESCRIPTION
**Pull request by ThomasCardin**
- Closes #39

### CODE REVIEW

**Review by vivalareda — APPROVED** [review]
looks good, iirc we fixed the lint-test with William last friday so should pass once that PR is approved

File: .github/workflows/azure-deployment.yml
Comment (rngadam) [inline .github/workflows/azure-deployment.yml]: confused as to why this is called azure-deployment but uses deploy-gcp?

**Review by rngadam — CHANGES_REQUESTED** [review]

File: .github/workflows/azure-deployment.yml
Comment (ThomasCardin) [reply .github/workflows/azure-deployment.yml]: Done

**Review by rngadam — CHANGES_REQUESTED** [review]
rework your commit history to link to issue and provide a meaningful commit message

**Comment by ThomasCardin** [comment]
> rework your commit history to link to issue and provide a meaningful commit message

For every single commit I make, you want me to link the issue? The PR is already linked to the issue (with the number and the title). When this is merged, the commit displayed on the main branch of this PR will contain the issue number and the title.

**Comment by ThomasCardin** [comment]
> rework your commit history to link to issue and provide a meaningful commit message

done @rngadam

**Comment by ThomasCardin** [comment]
@rngadam 

Considering that we frequently switch between cloud accounts (Azure, AWS, GCP), wouldn't it be better to simply push our images to our GitHub instead? It would allow us to stop constantly changing our credentials in the github secrets. I could take care of making the changes in the repositories if I have the green light.

**Comment by rngadam** [comment]
> Considering that we frequently switch between cloud accounts (Azure, AWS, GCP), wouldn't it be better to simply push our images to our GitHub instead? It would allow us to stop constantly changing our credentials in the github secrets. I could take care of making the changes in the repositories if I have the green light.

sound reasonable

**Comment by ThomasCardin** [comment]
> > Considering that we frequently switch between cloud accounts (Azure, AWS, GCP), wouldn't it be better to simply push our images to our GitHub instead? It would allow us to stop constantly changing our credentials in the github secrets. I could take care of making the changes in the repositories if I have the green light.
> 
> sound reasonable

Ok, then i'm closing this PR since we are now pushing images to the Github container registry (GCR)

**Pull request closed without merging**
